com.unity.render-pipelines.universal/Runtime/Tiling/SliceCullingJob.cs

```diff
@@ -0,0 +1,165 @@
+using System.Runtime.CompilerServices;
+using Unity.Burst;
+using Unity.Collections;
+using Unity.Jobs;
+using Unity.Mathematics;
+
+namespace UnityEngine.Rendering.Universal
+{
+    // Culls slices along one axis of the screen.
+    [BurstCompile]
+    unsafe struct SliceCullingJob : IJobFor
+    {
+        public float scale;
+        public float3 viewOrigin;
+        public float3 viewForward;
+        public float3 viewRight;
+        public float3 viewUp;
+
+        [ReadOnly]
+        public NativeArray<LightType> lightTypes;
+
+        [ReadOnly]
+        public NativeArray<float> radiuses;
+
+        [ReadOnly]
+        public NativeArray<float3> directions;
+
+        [ReadOnly]
+        public NativeArray<float3> positions;
+
+        [ReadOnly]
+        public NativeArray<float> coneRadiuses;
```

Comment:
Yea I agree on this, at least most of these properties are only used in this job. If we do not plan to re-usem in other jobs, I think it would be better to keep them in single structure. Keep in mind that NativeArray accesing is quite expensive in editor, because of all security checks.

Reply:
That's a fair point. I will look at reverting this to AoS in a later PR :)

Reply:
Regarding the data access, CPUs should be able to handle that amount of streams. I will need to ensure that this does not affect code gen.